server setup with google oauth logic

darkwhitenez opened the pull request:
Basic configuration of server with google signin logic

darkwhitenez committed:
server setup with google oauth logic

Basic configuration of server with google signin logic

mahboubii commented:
Good job, try to avoid adding static/frontend files, just focus on creating a neat API and documentation.

darkwhitenez committed:
better code organization

- separate directories for routes, controllers and models
- removed static files

darkwhitenez committed:
express-jwt added

Using express-jwt wrapper for more convenient approach to token handling
